modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -4,81 +4,60 @@ import ReactDOM from 'react-dom';
 import {
   createSubcomponent,
   createElemPropsHook,
-  StyledType,
   useWindowSize,
   useForkRef,
-  getTheme,
 } from '@workday/canvas-kit-react/common';
 import {usePopupModel, usePopupStack} from '@workday/canvas-kit-react/popup';
 import {keyframes} from '@emotion/react';
-import styled from '@emotion/styled';
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+import {Box, BoxProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {useModalModel} from './hooks';
+import {createStencil} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
 
 export interface ModalOverlayProps extends BoxProps {}
 
-const fadeIn = keyframes`
-  from {
-    background: none;
-  }
-  to {
-    background: rgba(0,0,0,0.65);
-  }
-`;
+const fadeIn = () => {
+  return keyframes({
+    from: {
+      background: 'none',
+    },
+    to: {
+      background: `rgba(0,0,0, ${system.opacity.overlay})`,
+    },
+  });
+};
 
-const Container = styled(Box)<StyledType>(
-  {
+const container = createStencil({
```

Comment:
```suggestion
const containerStencil = createStencil({
```
should it be like this?

Reply:
Yes it should. Updated.